src/components/NameList.js

```diff
@@ -0,0 +1,48 @@
+import { useState, useEffect } from "react";
+import ProfileCircle from "./profileCircle";
+import { get } from "../service/apiClient";
+
+function NameList() {
+  const [users, setUsers] = useState([]);
+  console.log(users[0]);
+
+  useEffect(() => {
+    const getUserInfo = async () => {
+      const res = await get(`users`);
+      setUsers([...users, res.data.users]);
```

Comment:
If you're getting all users here and will not again, you may as well just set it as such:
`setUsers(res.data.users)`
Then you don't need to to `users[0]` later, which should be avoided anyway